test/test_server.py

```diff
@@ -15,21 +15,20 @@
 """Test the server module."""
 
 import sys
-
-sys.path[0:0] = [""]
-
 from test import unittest
 
 from pymongo.hello import Hello
 from pymongo.server import Server
 from pymongo.server_description import ServerDescription
 
+sys.path[0:0] = [""]
+
```

Comment:
Can you undo this change? We do this before importing test/pymongo so that we can run the test file directly like `python test/test_server.py`.